src/containers/BookForm.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+// import PropTypes from 'prop-types';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+const defaultState = {
+  category: '',
+  title: '',
+  author: '',
+  pages: 0,
+  progress: 0,
+  summary: '',
+};
+
+/**
+ * Bookform
+ *
+ * Add books
+ */
+class BookForm extends Component {
+  constructor() { // (props) {
+    super(); // (props);
+    this.state = defaultState;
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(key, value) {
+    this.setState(prevState => ({
+      ...prevState,
+      [key]: value,
+    }));
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const formData = {
+      category: e.target.category.value,
+      title: e.target.title.value,
+      author: e.target.author.value,
+      pages: e.target.pages.value,
+      progress: e.target.progress.value,
+      summary: e.target.summary.value,
+    };
+    // eslint-disable-next-line no-console
+    console.log(formData);
```

Comment:
This should be a production code, please remove debugging statements. (`console.log`), even if that feature is not fully implemented, users should not see debugging messages, then you can remove unnecessarily `eslint-disable`

Reply:
Solved!

`console.log` is not anymore!:

https://github.com/mcastorena0316/Bookstore_CMS/blob/b833bc1e94a0d626112aa06353e5f49fefbbf10b/src/containers/BookForm.js#L36-L45